Chrome/bridge.js

```diff
@@ -12,6 +12,16 @@ var bridge_error_object;
 var xkit_storage = {};
 var bridge_ver = "2.1";
 
+// If chrome isn't defined, check for browser or msBrowser and use those instead. 
+if (typeof chrome === 'undefined') {
```

Comment:
this code shouldn't be in Chrome/bridge.js. Instead, you should probably make an Edge specific bridge, right?